### PR TITLE
fix(TDP-4257): include pendo snippet to application

### PR DESCRIPTION
**Pull request by ncomont**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4257

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

File: dataprep-webapp/src/app/services/utils/config/utils-analytics-account-service.spec.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  This source code is available under agreement available at
+  https://github.com/Talend/data-prep/blob/master/LICENSE
+
+  You should have received a copy of the agreement
+  along with this program; if not, write to Talend SA
+  9 rue Pages 92150 Suresnes, France
+
+  ============================================================================*/
+
+describe('Analytics account service', function () {
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/services/utils/config/utils-analytics-account-service.spec.js]: =>ES6

File: dataprep-webapp/src/app/services/utils/config/utils-analytics-account-service.spec.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    'use strict';
+
+    beforeEach(angular.mock.module('data-prep.services.utils'));
+
+    it('should set value by default', inject(function (analyticsAccount) {
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/services/utils/config/utils-analytics-account-service.spec.js]: =>ES6

File: dataprep-webapp/src/app/services/utils/config/utils-analytics-enabled-service.spec.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  This source code is available under agreement available at
+  https://github.com/Talend/data-prep/blob/master/LICENSE
+
+  You should have received a copy of the agreement
+  along with this program; if not, write to Talend SA
+  9 rue Pages 92150 Suresnes, France
+
+  ============================================================================*/
+
+describe('Analytics state service', function () {
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/services/utils/config/utils-analytics-enabled-service.spec.js]: =>ES6

File: dataprep-webapp/src/app/services/utils/config/utils-analytics-enabled-service.spec.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    'use strict';
+
+    beforeEach(angular.mock.module('data-prep.services.utils'));
+
+    it('should set value by default', inject(function (analyticsEnabled) {
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/services/utils/config/utils-analytics-enabled-service.spec.js]: =>ES6

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/565/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/565/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/571/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/571/)

**Pull request merged**